django_cache_manager/mixins.py

```diff
@@ -72,9 +68,16 @@ def invalidate_model_cache(self):
         Invalidate model cache by generating new key for the model.
         """
         logger.info('Invalidating cache for table {0}'.format(self.model._meta.db_table))
-        related_tables = set([rel.model._meta.db_table for rel in self.model._meta.get_all_related_objects()])
-        # temporary fix for m2m relations with an intermediate model, goes away after better join caching
-        related_tables |= set([field.rel.to._meta.db_table for field in self.model._meta.fields if issubclass(type(field), RelatedField)])
+        if django.VERSION >= (1, 8):
+            related_tables = set(
```

Comment:
It wasn't working with Django 1.10. I've just copied over the behaviour used for newer Django versions from https://github.com/Eugeny/django-cache-manager/blob/739eeafa38e45254903d8088a3749bfb51361155/django_cache_manager/models.py#L42